docs/products/metrics/howto/migrate-m3db-thanos.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Migrate Aiven for M3DB to Aiven for Metrics
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Migrate your Aiven for M3DB databases to Aiven for Thanos Metrics using the Aiven Console migration tool.
+
+## Compatibility considerations
+
+Review your current M3DB setup before migrating to Aiven for Metrics.
+
+- **Review database setup:** Examine your M3DB database's data structures, storage
+  patterns, and configurations. Identify any unique features, custom settings, and
+  specific configurations.
+- **API and query language compatibility:** Thanos Metrics uses PromQL, which is
```

Comment:
No, both use promql, so all existing queries will work as is most of the time. It is possible though that the same metric will have a slightly different name when written to Thanos directly as compared with its name in m3db.

For example, `prometheus_<metric-name>` in m3db might be called just `<metric-name>` in Thanos.
That is there will be two names for the same metric after migration:
  - `prometheus_<metric-name>{_source=m3db, labels...}` (for timeseries migrated from m3db)
  - `<metrci_name>{labels...}` (for timeseries written to thanos directly)
 
In this example, all queries will need to be adjusted accordingly. 

And again. all aiven-generated built-in dashboards will work out of the box.

Reply:
Update the point based on the inputs here.